api.js

```diff
@@ -0,0 +1,107 @@
+const hostComments = "https://webdev-hw-api.vercel.app/api/v2/olya-jacobs/comments";
+const hostLogin = "https://webdev-hw-api.vercel.app/api/user/login";
+const hostRegisterUser = "https://webdev-hw-api.vercel.app/api/user";
+
+export function addComment(token,name, text) {
+  return fetch(hostComments, {
+    method: "POST",
+    headers: {
+      Authorization: token,
+    },
+    body: JSON.stringify({
+      name: name,
+      date: new Date(),
+      text: text,
+      likes: 0,
+      isLiked: false,
+      forceError: true

```

Comment:
Апишка принимает только один ключ - text, остальные параметры игнорируются сервером. Можно остальные удалить
```suggestion
      text: text
```